client/src/modules/list/components/Items/ListItemDescription/ListItemDescription.jsx

```diff
@@ -0,0 +1,275 @@
+import React, { Fragment, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import _trim from 'lodash/trim';
+import _isEqual from 'lodash/isEqual';
+import Linkify from 'react-linkify';
+import Textarea from 'react-textarea-autosize';
+import classNames from 'classnames';
+
+import Preloader, {
+  PreloaderSize,
+  PreloaderTheme
+} from 'common/components/Preloader';
+import { RouterMatchPropType } from 'common/constants/propTypes';
+import { updateListItem } from '../model/actions';
+import { KeyCodes } from 'common/constants/enums';
+import { AbortPromiseException } from 'common/exceptions/AbortPromiseException';
+import { makeAbortablePromise } from 'common/utils/helpers';
+
+class ListItemDescription extends PureComponent {
+  pendingPromise = null;
+
+  constructor(props) {
+    super(props);
+
+    const { description } = this.props;
+    const trimmedDescription = description.trim();
+
+    this.state = {
+      isDescriptionUpdated: false,
+      descriptionInputValue: trimmedDescription,
+      isTextareaVisible: false,
+      isFocused: false,
+      pending: false
+    };
+
+    this.descriptionTextarea = React.createRef();
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    this.isDescriptionUpdated();
+    const { isTextareaVisible: previousIsTextareaVisible } = prevState;
```

Comment:
what does it mean ?
previous Is Textarea Visible? poprzednia jest textarea widoczna?